sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemClientBuilder.java

```diff
@@ -286,14 +286,14 @@ public DataLakeFileSystemClientBuilder setAnonymousAccess() {
     }
 
     /**
-     * Sets the {@link FileSystemEncryptionScope encryption scope} that is used to determine how file systems are
+     * Sets the {@link FileSystemEncryptionScopeOptions encryption scope} that is used to determine how file systems are
      * encrypted on the server.
      *
      * @param fileSystemEncryptionScope Encryption scope containing the encryption key information.
      * @return the updated DataLakeFileSystemClientBuilder object
      */
     public DataLakeFileSystemClientBuilder fileSystemEncryptionScope(
-        FileSystemEncryptionScope fileSystemEncryptionScope) {
+        FileSystemEncryptionScopeOptions fileSystemEncryptionScope) {
```

Comment:
nit: let's also rename the method and variable names to `fileSystemEncryptionScopeOptions`.